### PR TITLE
Support for gravatar

### LINKED ISSUE #418: Gravatar support for images
Based on [this comment](https://github.com/mitodl/micromasters/issues/394#issuecomment-221675266) we are going to implement Gravatar support for profile images.

We are also going to postpone the implementation of #394 and #179

### PR DESCRIPTION
**Pull request by giocalitri**
#### What are the relevant tickets?
closes #418 

#### What's this PR do?
Introduces support for gravatar images. It also removes the images URL from the profile

#### Where should the reviewer start?
`profiles/serializers.py`

#### How should this be manually tested?
You need to create an account with an email that has a gravatar image. Also test that for an email that has not a gravatar image, the default image is shown.

#### Any background context you want to provide?
To create a gravatar account go to https://en.gravatar.com

### CODE REVIEW

File: profiles/models.py [hunk 1]
```diff
@@ -122,10 +122,7 @@ class Profile(models.Model):
     )
 
     has_profile_image = models.BooleanField(default=False)
```
Comment (noisecapella) [inline profiles/models.py]: Do we still need this field?

File: profiles/util.py [hunk 2]
```diff
@@ -21,3 +23,39 @@ def split_name(name):
...
+        size (int): an integer representing the image size to be requested
+    Returns:
+        string: a string representi the gravatar url
+    """
+    base_gravatar_url = "https://www.gravatar.com/avatar/"
+    default_image_url = "https://s3.amazonaws.com/odl-micromasters-production/avatar_default.png"
+
+    # return the gravatar image
+    md5_hash = hashlib.md5()
+    md5_hash.update(user_email.lower().encode())
```
Comment (noisecapella) [inline profiles/util.py]: It looks like the official docs include a trim function: https://en.gravatar.com/site/implement/hash/
Can we add it here, even if it's not strictly necessary for our case?

**Comment by noisecapella** [comment]
Functionality works fine

**Comment by noisecapella** [comment]
Code looks good :+1:

File: profiles/util.py (hunk 2 above)
Comment (giocalitri) [inline profiles/util.py]: you are right: will do

File: profiles/models.py (hunk 1 above)
Comment (giocalitri) [inline profiles/models.py]: I thought it might be useful in the future... no strong opinion

Comment (noisecapella) [inline profiles/models.py]: No strong opinion here either

**Pull request merged**